handshake_manager.go

```diff
@@ -191,13 +191,13 @@ func (c *HandshakeManager) handleOutbound(vpnIp iputil.VpnIp, f udp.EncWriter, l
 	}
 }
 
-func (c *HandshakeManager) AddVpnIp(vpnIp iputil.VpnIp) *HostInfo {
-	hostinfo := c.pendingHostMap.AddVpnIp(vpnIp)
-	// We lock here and use an array to insert items to prevent locking the
```

Comment:
I think if we remove this then we should add the locking or rename the function to `UnlockedAddVpnIp`

Reply:
This comment is outdated and from before the lock was moved to inside of the OutboundHandshakeTimer itself. Now we can finally delete this old and wrong comment!